src/layouts/pt-br/docsbr.astro

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This also would be removed due to it not being language-agnostic in layout definition, right?

Reply:
I thought I removed these files, I'll remove